includes/functions/llms.functions.favorite.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * LifterLMS Favorite Functions
+ *
+ * @package LifterLMS/Functions
+ *
+ * @since [version]
+ * @version [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Get Favorites Count.
+ *
+ * @since [version]
+ *
+ * @param WP_Post|int|false $object_id  Lesson post object or id. If `false` uses the global `$post` object.
+ * @return int
+ */
+function get_total_favorites( $object_id = false ) {
```

Comment:
https://github.com/gocodebox/lifterlms/blob/trunk/docs/coding-standards.md#functions--variables

`llms_` prefix is required.
Also we're getting an object total favorites right, so maybe `llms_get_object_total_favorites()`?
Also you're getting an optional parameter and you say that the global `$post` object will be used when not supplied, but I don't see this.

Reply:
fixed.

Reply:
@actuallyakash 
fixed? but where?
can you push the fixes before saying fixed? it's confusing if you say "fixed" but I don't see the fix.

Reply:
just a minute, some CS fixes stopped me from pushing. Gonna push in few

Reply:
I understand sure, I'm just asking you to follow this process next time:
- fix it
- push the fix
- notice the reviewer of the fix
:D

Reply:
got it. 👍